var/spack/repos/builtin/packages/amdfftw/package.py

```diff
@@ -129,6 +56,11 @@ def configure(self, spec, prefix):
 
         # Variants that affect every precision
         if '+openmp' in spec:
+            # Note: Apple's Clang does not support OpenMP.
+            if spec.satisfies('%clang'):
+                ver = str(self.compiler.version)
+                if ver.endswith('-apple'):
+                    raise InstallError("Apple's clang does not support OpenMP")
```

Comment:
We recently split LLVM Clang and Apple Clang into separate compilers. The `conflicts` above should be used instead of this check.